packages/app-mobile/package.json

```diff
@@ -73,19 +73,19 @@
   "devDependencies": {
     "@babel/core": "7.16.0",
     "@babel/runtime": "7.16.3",
-    "@codemirror/commands": "6.0.0",
-    "@codemirror/lang-cpp": "6.0.1",
-    "@codemirror/lang-html": "6.0.0",
-    "@codemirror/lang-java": "6.0.0",
-    "@codemirror/lang-javascript": "6.0.0",
-    "@codemirror/lang-markdown": "6.0.0",
-    "@codemirror/lang-php": "6.0.0",
-    "@codemirror/lang-rust": "6.0.0",
-    "@codemirror/language": "6.0.0",
-    "@codemirror/legacy-modes": "6.1.0",
-    "@codemirror/search": "6.0.0",
-    "@codemirror/state": "6.0.0",
-    "@codemirror/view": "6.0.0",
+    "@codemirror/commands": "6.1.2",
+    "@codemirror/lang-cpp": "6.0.2",
+    "@codemirror/lang-html": "6.1.3",
+    "@codemirror/lang-java": "6.0.1",
+    "@codemirror/lang-javascript": "6.1.1",
+    "@codemirror/lang-markdown": "6.0.5",
+    "@codemirror/lang-php": "6.0.1",
+    "@codemirror/lang-rust": "6.0.1",
+    "@codemirror/language": "6.3.0",
+    "@codemirror/legacy-modes": "6.2.0",
+    "@codemirror/search": "6.2.2",
+    "@codemirror/state": "6.1.3",
+    "@codemirror/view": "6.4.2",
```

Comment:
Is it expected that they all have slightly different version number? Could we set everything to eg 6.3.0? (Assuming CodeMirror is managed as a monorepo with all packages having the same number)

Reply:
CodeMirror isn't managed as a monorepo — they all have different version numbers (try searching for the individual packages on https://yarnpkg.com/).

Reply:
Ok thanks for clarifying, we can merge then